src/main/java/hw8/beans/DataSet.java

```diff
@@ -0,0 +1,100 @@
+
+package hw8.beans;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class DataSet {
```

Comment:
This is completely prohibited to work with data in this way. Imagine the situation if the data set will be changed.

Reply:
Besides, this class is useless.

Reply:
removed